### PR TITLE
BookA-SpatialGeoObsSensor

### PR DESCRIPTION
**Pull request by larswik**
# Pset: Pset_SpatialGeoObsSensor

## Description: 

Properties describing sensors used for geoscience observations.

| Property | Datatype | Description |
|----------|----------|-------------|
|PlacementType|PEnum_SensorPlacementType|Type of investigation to install sensors.|
|SensorType|IfcLabel|Type and model of a sensor - Not part of the Pset/Qto.|
|SensorName|IfcLabel|Identifier of a sensor like No. and name. - Not part of the Pset/Qto.|
|MeasurementStartTime|IfcDateTime|Measurement/Monitoring period or point in time - start.|
|MeasurementEndTime|IfcDateTime|Measurement/Monitoring period or point in time - end.|

## Enumeration: Penum_SensorPlacementType

| Constant | Description |
|----------|-------------|
|MONITORING| |
|SURVEY| |

### CODE REVIEW

**Comment by patmclarin** [comment]
please update description to: Properties describing sensors used for geoscience observations

File: IFC4x3/Properties/m/MeasurementEndTime_2h0TtkB$f46PgH3YITllt2/DocProperty.xml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (SergejMuhic) [inline IFC4x3/Properties/m/MeasurementEndTime_2h0TtkB$f46PgH3YITllt2/DocProperty.xml]: How about using the existing `StartDate` and `EndDate`? Of course making their descriptions a bit more generic.

File: IFC4x3/Properties/p/PlacementType_3Fx8gNev9BdO6QiED3BT8F/Documentation.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Type of investigation to install sensors.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/p/PlacementType_3Fx8gNev9BdO6QiED3BT8F/Documentation.md]: This does not seem to match with the property name,

**Review by SergejMuhic — COMMENTED** [review]
Constant and enumeration descriptions.

File: IFC4x3/Properties/m/MeasurementEndTime_2h0TtkB$f46PgH3YITllt2/DocProperty.xml (hunk 1 above)
Comment (larswik) [reply IFC4x3/Properties/m/MeasurementEndTime_2h0TtkB$f46PgH3YITllt2/DocProperty.xml]: I think that there is semantics in the name and StartDate/EndDate indicates Date and not DateTime (even though description/and datatype indicates DateTime) so imho we should keep the current naming (MeasurementStartTime/*EndTime)

File: IFC4x3/Properties/p/PlacementType_3Fx8gNev9BdO6QiED3BT8F/Documentation.md (hunk 2 above)
Comment (larswik) [reply IFC4x3/Properties/p/PlacementType_3Fx8gNev9BdO6QiED3BT8F/Documentation.md]: > This does not seem to match with the property name,

Ping @j-weil or @patmclarin (could you also provide descriptions for the enum constants?)

**Review by SergejMuhic — APPROVED** [review]

**Pull request merged**